modules/table/src/testFixtures/java/org/apache/ignite/internal/table/TableTestUtils.java

```diff
@@ -239,6 +239,7 @@ public static int getTableIdStrict(CatalogService catalogService, String tableNa
         return getTableStrict(catalogService, tableName, timestamp).id();
     }
 
+
```

Comment:
Horrible, disgusting mistake. Please remove this empty line.

Reply:
fixed